etstool.py

```diff
@@ -414,12 +414,17 @@ def api_docs(edm, runtime, toolkit, environment):
     click.echo("Done regenerating API docs")
 
     os.chdir("docs")
-    commands = ["{edm} run -e {environment} -- make html"]
+    command = (
+        "edm run -e {environment} -- sphinx-build -b html "
```

Comment:
The original code uses `{edm}`, it'd be good to be consistent. That said, I don't know what motivates the option to use a different EDM.
```suggestion
        "{edm} run -e {environment} -- sphinx-build -b html "
```